src/cloud/ws-connection.ts

```
@@ -0,0 +1,211 @@
+        this.activeBinds.delete(state.id);
+      },
+      start: (controller) => {
+        this.onMsg<S>((msg) => {
+          if (MsgIsError(msg)) {
+            controller.enqueue(msg);
+            return;
+          }
+          if (!MsgIsTid(msg, req.tid)) {
+            return;
+          }
+          if (opts.waitFor && opts.waitFor(msg)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)